<ui version="4.0" >
 <class>VSSequenceView</class>
 <widget class="QWidget" name="VSSequenceView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Sequence:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3" >
      <widget class="QLineEdit" name="sequence_name" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="duration" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Num. blocks:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QLineEdit" name="num_blocks" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="STGlobalView" native="1" name="global_view" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>STGlobalView</class>
   <extends>QWidget</extends>
   <header>stglobalview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
